integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -230,16 +217,84 @@ public static void deleteNamespace(String namespace) throws Exception {
   }
 
   /**
-   * create a namespace.
-   *
-   * @param namespace - namespace to create
-   * @throws Exception - if any error occurs
+   * Create a namespace.
+   * 
+   * @param namespace namespace to create
+   * @throws Exception if any error occurs
    */
   public static void createNamespace(String namespace) throws Exception {
     if (!namespace.equalsIgnoreCase("default")) {
-      String command = "kubectl create ns " + namespace;
-      logger.info("Running " + command);
-      TestUtils.execOrAbortProcess(command);
+      String cmd1 = "kubectl delete ns " + namespace + " --ignore-not-found";
+      logger.info("Running " + cmd1);
+      TestUtils.execOrAbortProcess(cmd1, true);
+      String cmd2 = "kubectl create ns " + namespace;
+      logger.info("Running " + cmd2);
+      TestUtils.execOrAbortProcess(cmd2, true);
+    }
+  }
+  
+  /**
+   * Start DB instance, create Oracle rcu pod and load database schema in the specified namespace.
+   * 
+   * @param scriptsDir directory of scripts
+   * @param dbPort NodePort of DB
+   * @param dbUrl URL of DB
+   * @param rcuSchemaPrefix rcu SchemaPrefixe
+   * @param namespace namespace where DB and RCU schema are going to start
+   * @throws Exception if any error occurs when setting up RCU database
+   */
+  public static void setupRcuDatabase(String scriptsDir, int dbPort, String dbUrl, 
+      String rcuSchemaPrefix, String namespace) throws Exception {  
+    
+    createNamespace(namespace);
+    createDockerRegistrySecret(namespace);
+    /*
```

Comment:
Delete commented out code.

Reply:
Done. 
(Sorry, I just saw this review comment.  Since I don't know if we are going to need this kind of change in the old integration test  I just created a draft PR without code cleanup.)